docs/api.rst

```diff
@@ -733,6 +733,12 @@ section generates a new :class:`URL` instance.
       Support subclasses of :class:`int` (except :class:`bool`) and :class:`float`
       as a query parameter value.
 
+.. method:: URL.without_query_params(*keys)
+
+   Return a new URL whose *query* part does not contain specified keys.
+
+   It does nothing if none of specified keys are present in the query.
```

Comment:
We should add a `.. versionadded::` directive right before merging.

Here's an example that I'd postpone committing until we know the version this will target.
```suggestion
   It does nothing if none of specified keys are present in the query.

   .. versionadded:: 1.10.0
```